models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php

```diff
@@ -415,4 +415,71 @@ public function loadLazyBrickField(DataObject\Objectbrick\Data\AbstractData $ite
             }
         }
     }
+
+    /**
+     * @internal trigger deprecation error when a relation is passed multiple times, remove in Pimcore 7
+     *
+     * @param array|null $data
+     * @param DataObject\Concrete|DataObject\Localizedfield|DataObject\Objectbrick\Data\AbstractData|\Pimcore\Model\DataObject\Fieldcollection\Data\AbstractData $object
+     * @param array $params
+     *
+     * @return array
+     *
+     * @throws \Exception
+     */
+    public function filterMultipleAssignments($data, $object, $params)
```

Comment:
Can we call it $container instead of $object from now on?

Reply:
done!